env/env_test.cc

```diff
@@ -1103,6 +1116,11 @@ TEST_F(EnvPosixTest, PositionedAppend) {
   options.use_direct_writes = true;
   options.use_mmap_writes = false;
   IoctlFriendlyTmpdir ift;
+  if (!ift.is_supported()) {
+    ROCKSDB_GTEST_BYPASS(
+        "FS_IOC_GETVERSION is not supported by the filesystem");
+    return;
+  }
```

Comment:
Is `FS_IOC_GETVERSION` strictly needed for this test or is the use of `IoctlFriendlyTmpdir` excessive?

Reply:
it's actually because of direct_io is enabled for these test which shm doesn't support. Added `SetupSyncPointsToMockDirectIO()` for these tests.